File: c/bee1046.c
#include <stdio.h>

int main() {
    int inicio, final, tempo;
    scanf("%d %d", &inicio, &final);

    tempo = final - inicio;
    if(tempo <= 0)
        tempo += 24;

    printf("O JOGO DUROU %d HORA(S)\n", tempo);

    return 0;
}

File: java/bee1046.java
import java.util.Scanner;

public class bee1046 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int horaIni = scanner.nextInt();
        int horaFim = scanner.nextInt();
        int duracao = 0;

        if(horaIni < horaFim) {
            duracao = horaFim - horaIni;
        } else {
            duracao = (horaFim + 24) - horaIni;
        }

        System.out.println("O JOGO DUROU " + duracao + " HORA(S)");
        scanner.close();
    }
}
